x-pack/plugins/apm/public/components/app/service_details/service_icons/index.test.tsx

```diff
@@ -51,6 +52,12 @@ function Wrapper({ children }: { children?: ReactNode }) {
 }
 
 describe('ServiceIcons', () => {
+  beforeEach(() => {
+    jest.spyOn(useThemeHook, 'useTheme').mockReturnValue({
+      eui: {},
```

Comment:
This is causing failures because ```Type '{}' is missing the following properties from type '{ paddingSizes: { xs: string; s: string; m: string; l: string; xl: string; }; avatarSizing: { s: { size: string; 'font-size': string; }; m: { size: string; 'font-size': string; }; l: { size: string; 'font-size': string; }; xl: { ...; }; }; ... 319 more ...; euiZToastList: number; }': paddingSizes, avatarSizing, euiBadgeGroupGutterTypes, euiBreadcrumbSpacing, and 318 more.``` Any ideas what should be defined here? cc @sqren

Reply:
@smith Thanks - I've replaced the hook and used `EuiThemeProvider`. Type checks are passing now 👍